### ISSUE TITLE
Entry viewer does not reports time in timezone set by my desktop

### ISSUE THREAD

**Issue by rachellann** [CalderaWP/Caldera-Forms#1344]
Is it possible to set timezone sync to my desktop or local timezone?

**Comment by Shelob9** [comment]
@rachellann Are you refering to the entry viewer in the admin or the front-end entry viewer?

**Comment by rachellann** [comment]
Yes, I'm referring to the front-end entry viewer. It doesn't sync to my time zone.
